humans/athayde_john/details.json

```diff
@@ -1,14 +1,14 @@
 {
-  "title": "",
+  "title": "VP of Design / Owner",
   "slug": "john-athayde",
   "role": [
     "Speaker"
   ],
-  "twitter": "",
-  "facebook": "",
-  "linkedin": "",
-  "personalWebsite": "",
-  "companyWebsite": "",
+  "twitter": "boboroshi",
+  "facebook": "facebook.com/johnathayde",
+  "linkedin": "https://www.linkedin.com/in/johnathayde/",
+  "personalWebsite": "www.sfumatofarm.com",
+  "companyWebsite": "www.meticulous.com",
```

Comment:
@boboroshi thanks :) only thing is that these need to be full urls and escape the slashes: `http:\/\/www.meticulous.com`